### PR TITLE
Command Line Interface

### PR DESCRIPTION
**Pull request by chriseppstein**
The CLI is useful for testing css-blocks or doing basic compilation to BEM classes.

### CODE REVIEW

File: packages/@css-blocks/cli/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import { BlockFactory, CssBlockError, Preprocessors } from "@css-blocks/core";
```
Comment (amiller-gh) [inline packages/@css-blocks/cli/src/index.ts]: Add comments to this file then looks good to me.

File: packages/@css-blocks/cli/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import chalk = require("chalk");
+import fs = require("fs");
```
Comment (amiller-gh) [inline packages/@css-blocks/cli/src/index.ts]: Doesn't `fs-extra` have a promisified version of writeFile you can use?

File: packages/@css-blocks/cli/src/index.ts (hunk 1 above)
Comment (chriseppstein) [reply packages/@css-blocks/cli/src/index.ts]: ✅ Comments added.

File: packages/@css-blocks/cli/src/index.ts (hunk 2 above)
Comment (chriseppstein) [reply packages/@css-blocks/cli/src/index.ts]: yep! Using that now.

**Pull request merged**